### ISSUE TITLE
Sensitive Roadside Areas- Cultural Resources

### ISSUE THREAD

**Issue by gfuellingwa** [wsdot-eso/Environmental-Information#126]
**Description:**
**Category:**
**Workspace:**
**% Complete:** 0%

Data needs updated from what is on the workbench. Data Steward needs to be finalized

**Comment by jesse-WSDOT** [comment]
Sent email to Erin L. about dataset updates

**Comment by jesse-WSDOT** [comment]
RE:
Thanks! When it was initially created we used a **### 500ft buffer on the road layer to identify archaeological sites and national register listed properties in proximity to our roadways.** The buffered road segments with intersecting sites were then merged to assign attributes from the site layer to the road segment as a way to identify sensitive areas without giving away the exact site locations. Sites could be located within or adjacent to the road prism and should trigger maintenance to contact our program for further steps. That was the intent and my understanding of how things were developed- but that was many years ago, so it has likely been modified and streamlined. Let me know how I can help.
Thanks,
Erin
